### ISSUE TITLE
No 'version' key in the manifest file

### ISSUE THREAD

**Issue by FlashGordon7** [garbled1/balboa_homeassistan#19]
[homeassistant.loader] No 'version' key in the manifest file for custom integration 'balboa'. As of Home Assistant 2021.6, this integration will no longer be loaded. Please report this to the maintainer of 'balboa'

**Comment by yetdog** [comment]
Bumping this, hoping to get some visibility. I love this integration so hope it stays current!

**Comment by FlashGordon7** [comment]
Yes, let's hope it get fixed.
If not, it's easy to fix, just locate custom_components/balboa/manifest.json
then change:
```
  "codeowners": [
    "@garbled1",
    "@natekspencer"
  ]
}
```
To this
```
  "codeowners": [
    "@garbled1",
    "@natekspencer"
  ],
  "version": "0.6"
}
```

**Comment by reicharb** [comment]
FlashGordon7 thank you.  That worked.

**Comment by garbled1** [comment]
Sorry for the delay, fixed in 0.6.1
